CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.23+1
```

Comment:
It's not worth publishing this version, since it doesn't have any user-visible changes.